katello/katello-change-hostname

```diff
@@ -9,7 +9,14 @@ end
 
 require File.join(UTILS_PATH, 'hostname-change.rb')
 
-hostname_change = KatelloUtilities::HostnameChange.new("Foreman Proxy", "Foreman Proxies", "foreman-proxy-content")
+options = {
+ proxy: "Foreman Proxy",
+ plural_proxy: "Foreman Proxies",
+ proxy_hyphenated: "foreman-proxy-content",
+ plugin: "katello"
```

Comment:
I think I might prefer this being `command` or `command_name` rather than plugin. Plugin tends to evoke a lot of other thoughts and this isn't really a plugin system.